Corona-Warn-App/src/main/java/de/rki/coronawarnapp/ui/settings/SettingsNotificationFragment.kt

```diff
@@ -75,10 +81,18 @@ class SettingsNotificationFragment : Fragment() {
         updateRiskNotificationSwitch.setOnClickListener {
             settingsViewModel.toggleNotificationsRiskEnabled()
         }
+        // Additional click target to toggle switch
+        updateRiskNotificationRow.setOnClickListener {
+            settingsViewModel.toggleNotificationsRiskEnabled()
```

Comment:
The row has to check if the switch is disabled to be future proof and be consistent with settings tracing.

` if (updateRiskNotificationRow.isEnabled) settingsViewModel.toggleNotificationsRiskEnabled()`

Reply:
implemented in commit 0dc17a1